dss/src/processing.py

```diff
@@ -204,13 +200,13 @@ def get_run_score(run_dir, params):
     param_scores = {}
     for param in model_analysis_params:
         param_value = get_run_parameter_value(param['name'], contents)
-        param_score = calc_param_score(param_value, float(param['target']), float(param['score_step']), int(param['desired_direction']))
+        param_score = calc_param_score(param_value, float(param['target']), float(param['score_step']))
```

Comment:
If the algorithm doesn't normalize the weights, there's no need to capture the weight and the score separately, this can be moved into the calc_param_score function.

btw, please confirm that the new algorithm behavior is as you intended. Weights close to 0 will cause very high scores, even for very small differences from the target....